packages/launchpad/src/setup/SelectFwOrBundler.vue

```diff
@@ -34,6 +34,7 @@
         {{ selectedOptionName }}
       </span>
       <AlphaLabel v-if="selectedFrameworkOptionObject?.supportStatus === 'alpha'" />
+      <CommunityLabel v-if="selectedFrameworkOptionObject?.supportStatus === 'community'" />
```

Comment:
We could refactor `AlphaLabel` to be generic

Reply:
Incoming style change for the color, but might be a neat refactor here, like `FrameworkLabel`